specification/enterprisesupport/resource-manager/Microsoft.EnterpriseSupport/preview/2022-07-01/enterpriseSupport.json

```diff
@@ -0,0 +1,809 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Enterprise Billing Resource Provider",
+    "description": "",
```

Comment:
New namespaces need to be onboarded to RPaaS. Please send a mail to armapireview alias explaining why this service is being onboarded as a non-RPaaS RP.